README.md

```diff
@@ -50,3 +50,18 @@ It also shows how you can modify your pre-built Stratux US version to run the EU
 
 If you want to customize beyond that, or have different needs, you can find a full list of supported hardware/attachments [here](https://github.com/stratux/stratux/wiki/Supported-Hardware).
 
+## Developing
+
+See [Development.md](docs/Development.md) for details and information.
+
+## Docs in repository vs. in wiki
+
+At present, user facing documentation is stored in the wiki because the wiki is:
```

Comment:
"Hey look, I just found the Stratux project on the internet. I want to build an ADS-B receiver, that's cool. I can't wait to see there precise reasoning why they want dev docs in the repo and user docs in the Wiki"